### ISSUE TITLE
FYI: dlang-community / grpc-d works better

### ISSUE THREAD

**Issue by mw66** [huntlabs/hunt-grpc#35]
Just FYI:

There is another grpc-d lib:

https://github.com/dlang-community/grpc-d

and I have tried the server side, it works more reliable than this grpc-dlang library;
also it can talk to both Python and Dart client easily, e.g:

https://github.com/dlang-community/grpc-demo
https://github.com/dlang-community/grpc-d-compiler
https://github.com/dlang-community/grpc-d-interop

(I fixed a few things, so please use the dlang-community fork version of the libs.)

Basically, it's a D wrapper of Google's own libgrpc, not a home-grown library like this one, so we are using the same underlying libs from Google.

D has a relative small community, maybe we should join the effort to maintain one single quality library, instead of re-invent the wheels, unless you are confident you can do better than Google's own libs.

Cheers.
